client/src/components/player/PlayerPlot.tsx

```diff
@@ -31,15 +31,15 @@ const PlotItem = ({clue}: {clue: PlayerClue}) => {
                 );
             } else {
                 return (
-                        <Container fluid>
+                        <Container fluid style={{marginBottom: '37px'}}>
                             <ListGroup.Item>
                                 {clueSolveTimeAnchor}
                                 <Row style={{justifyContent: 'center', display: 'flex'}}>                                
                                     {solvedPlot.map(content => <AdditionalContent key={content.contentId} content={content} />)}
                                 </Row>
                                 <Row style={{justifyContent: 'center', display: 'flex'}}>
                                     <LinkContainer to={`/player/clue/${clue.tableOfContentId}`} style={{ color: "#FFFFFF" }}>
-                                        <Button><FaPuzzlePiece /> Puzzle</Button>
+                                        <Button>{(clue.submittableType === "Puzzle") && <FaPuzzlePiece />}{(clue.submittableType === "LocUnlock") && <FaLocationArrow />}{clue.submittableTitle}</Button>
```

Comment:
As a nitpick, it seems like we could probably have a function that returns a glyph based on submittable type since we're repeating this in 3 places.